libraries/lib-wave-track/WaveTrack.h

```diff
@@ -476,6 +474,17 @@ class WAVE_TRACK_API WaveTrack final
     */
    void Trim(double t0, double t1) /* not override */;
 
+   /*
+    * @param interval Entire track is rendered if nullopt, else only samples
+    * within [interval->first, interval->second), in which case clips are split
+    * at these boundaries before rendering - if rendering is needed.
+    *
+    * Precondition as for Clone
+    * @pre `!interval.has_value() || interval->first <= interval->second`
+    */
```

Comment:
When a zero duration interval is given, will there be a split there or will nothing happen?   Comment that

Reply:
It's implied: `in which case clips are split at these boundaries before rendering - if rendering is needed` -> empty interval, no rendering needed.

Reply:
So I understand that means, yes there is a split?

Reply:
No, there is no split. I didn't realize this was so unclear - will clarify the comment when I get the chance.